redis/src/test/scala/zio/redis/HashSpec.scala

```diff
@@ -0,0 +1,205 @@
+package zio.redis
+
+import zio.Chunk
+import zio.test._
+import zio.test.Assertion._
+
+trait HashSpec extends BaseSpec {
+
+  val hashSuite =
+    suite("hash")(
+      suite("hSet, hGet, hGetAll and hDel")(
+        testM("set followed by get") {
+          for {
+            hash   <- uuid
+            field  <- uuid
+            value  <- uuid
+            _      <- hSet(hash)((field, value))
+            result <- hGet(hash, field)
+          } yield assert(result)(isSome(equalTo(value)))
+        },
+        testM("set multiple fields for hash") {
+          for {
+            hash   <- uuid
+            field1 <- uuid
+            field2 <- uuid
+            value  <- uuid
+            result <- hSet(hash)((field1, value), (field2, value))
+          } yield assert(result)(equalTo(2L))
+        },
+        testM("get all fields for hash") {
+          for {
+            hash   <- uuid
+            field1 <- uuid
+            field2 <- uuid
+            value  <- uuid
+            _      <- hSet(hash)((field1, value), (field2, value))
+            result <- hGetAll(hash)
+          } yield assert(Chunk.fromIterable(result.values))(hasSameElements(Chunk(value, value)))
+        },
+        testM("delete field for hash") {
```

Comment:
The scope of this test is a bit broad. Let's remove `added` from the "equation".